Mathlib/MeasureTheory/Function/LpSpace/Basic.lean

```diff
@@ -911,4 +911,29 @@ theorem meas_ge_le_mul_pow_enorm (f : Lp E p μ) (hp_ne_zero : p ≠ 0) (hp_ne_t
   (ENNReal.ofReal_toReal (eLpNorm_ne_top f)).symm ▸
     meas_ge_le_mul_pow_eLpNorm_enorm μ hp_ne_zero hp_ne_top (Lp.aestronglyMeasurable f) hε (by simp)
 
+section Star
+
+variable {R : Type*} [NormedAddCommGroup R] [StarAddMonoid R] [NormedStarGroup R]
+
+protected noncomputable instance {p : ℝ≥0∞} : Star (Lp R p μ) where
+  star f := ⟨star (f : α →ₘ[μ] R),
+    by simpa [Lp.mem_Lp_iff_eLpNorm_lt_top] using Lp.eLpNorm_lt_top f⟩
+
+lemma coeFn_star {p : ℝ≥0∞} (f : Lp R p μ) : (star f : Lp R p μ) =ᵐ[μ] star f :=
+    (f : α →ₘ[μ] R).coeFn_star
+
+noncomputable instance {p : ℝ≥0∞} : InvolutiveStar (Lp R p μ) where
+  star_involutive f := by
+    ext
+    filter_upwards
+    exact congrFun (congrArg AEEqFun.cast <| star_involutive f.1)
```

Comment:
Can you simplify this by using `congrm(...)` instead? That's generally a nicer way to build terms like this.

Reply:
Not familiar with this. I was hoping there would be a way to avoid using `filter_upwards` here and use the `AEEqFun` version of `star_involutive`. Will `congrm` do this? I'm not sure what the holes here should be. Maybe we should be peeling of a single coercion to get an equality at the AEEqFun level?

Reply:
OK. Got to you on Zulip. Will try this soon.